provisioning/service/src/Config/TpmAttestation.cs

```diff
@@ -59,8 +59,8 @@ public string EndorsementKey
             private set
             {
                 /* SRS_TPM_ATTESTATION_21_001: [The EndorsementKey setter shall throws ArgumentException if the provided 
-                                                endorsementKey is null or invalid.] */
-                ParserUtils.EnsureBase64String(value);
+                                                endorsementKey is null or white space.] */
+                ParserUtils.EnsureNotNullOrWhiteSpaceString(value);
```

Comment:
`string` has this check. This is overly complicated and instead removes code clarity.